src/test/java/com/marizueva/laboratory/hw3/fluentpageobject/pages/CommonPageElements.java

```diff
@@ -0,0 +1,195 @@
+    WebElement subMenuFromHeaderTableWithPagesItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']"
+            + "/li/a[@href='different-elements.html']")
+    WebElement subMenuFromHeaderDifferentElementsItem;
+
+    @FindBy(css = "ul.sidebar-menu.left >li[index='3']")
+    WebElement leftServiceMenu;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']//span[text()='Support']")
+    WebElement subMenuFromLeftSupportItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']//span[text()='Dates']")
+    WebElement subMenuFromLeftDatesItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']//span[text()='Search']")
+    WebElement subMenuFromLeftSearchItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']//a[@href='complex-table.html']")
+    WebElement subMenuFromLeftComplexTableItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']//a[@href='simple-table.html']")
+    WebElement subMenuFromLeftSimpleTableItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']//a[@href='user-table.html']")
+    WebElement subMenuFromLeftUserTableItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']"
+            + "//a[@href='table-pages.html']")
+    WebElement subMenuFromLeftTableWithPagesItem;
+
+    @FindBy(xpath = "//div[@name='navigation-sidebar']"
+            + "//a[@href='different-elements.html']")
+    WebElement subMenuFromLeftDifferentElementsItem;
+
+    public ServiceDifferentElementPage getServiceDifferentElementPage() {
+        return serviceDifferentElementPage;
+    }
+
+    public CommonPageElements(WebDriver driver) {
+        super(driver);
+
+    }
+
+    public WebElement getLeftSection() {
+        return leftSection;
+    }
+
+    public WebElement getFooter() {
+        return pageFooter;
+    }
+
+
+    public CommonPageElements clickServiceItemFromHeader() {
+        headerItemService.click();
+        return this;
+    }
+
+    public WebElement getSubMenuServiceItemFromHeader(ServicesSubCategories subCategories) {
+        switch (subCategories) {
+            case SUPPORT:
+                return subMenuFromHeaderSupportItem;
+            case DATES:
+                return subMenuFromHeaderDatesItem;
+            case SEARCH:
+                return subMenuFromHeaderSearchItem;
+            case COMPLEX_TABLE:
+                return subMenuFromHeaderComplexTableItem;
+            case SIMPLE_TABLE:
+                return subMenuFromHeaderSimpleTableItem;
+            case USER_TABLE:
+                return subMenuFromHeaderUserTableItem;
+            case TABLE_WITH_PAGES:
+                return subMenuFromHeaderTableWithPagesItem;
+            case DIFFERENT_ELEMENT:
+                return subMenuFromHeaderDifferentElementsItem;
+            default:
+                throw new NotFoundException("element not found:" + subCategories);
+        }
+    }
```

Comment:
Why do you return `WebElement` here instead of the page?

Reply:
fixed